src/styles/main.css

```diff
@@ -171,3 +171,13 @@ input.rbt-input-main {
 .editor-navtabs a {
     color: #2F2424;
 }
+
+/** For lookup menu headers **/
+.dropdown-header {
+	font-weight:800;
+	color:#333;
```

Comment:
I think we should add a background color, perhaps `background-color: lightgray;`?

Reply:
Added background-color #d3d3d3 and updated the spoof file.